tests/libcxx/expected_results.txt

```diff
@@ -591,6 +584,18 @@ std/depr/depr.c.headers/stddef_h.compile.pass.cpp:1 FAIL
 
 # *** LIKELY BOGUS TESTS ***
 
+# A moved-from flat_map is not empty, which doesn't seem to be required
+std/containers/container.adaptors/flat.map/flat.map.cons/move_alloc.pass.cpp FAIL
+std/containers/container.adaptors/flat.multimap/flat.multimap.cons/move_alloc.pass.cpp FAIL
+
+# A moved-from flat_map is not meeting the first invariant, but this seems to also be a specification issue: 
+# [flat.map.defn] doesn't declare a move constructor, which seems to imply that moving from a flat_map should behave
+# as if the move constructor were implicitly defined, i.e. it can't restore the invariants

```

Comment:
I think we should mention WG21-P3567 here.

Reply:
Wow, the paper still asks to move the comparator

Reply:
I expanded the explanation to mention WG21-P3567 in two groups of failures, please check